app/views/attachments/index.html.erb

```diff
@@ -0,0 +1,32 @@
+<% content_for :back_link, render_back_link(href: document_path(@edition.document)) %>
+<% content_for :title, t("attachments.index.title", title: @edition.title_or_fallback) %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <p class="govuk-body">
+      <%= t("attachments.index.upload.description_govspeak") %>
+    </p>
+
+    <%= render "govuk_publishing_components/components/button", {
+      text: t("attachments.index.upload.title"),
```

Comment:
Can we double-check this against the prototype/design? I think we're also missing an `else` clause for when we have no featured attachments.